keyboards/ibm_6746/ibm_6746.h

```diff
@@ -0,0 +1,47 @@
+/* Copyright 2023 Justin (@Justin-sandwich)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+//  This is what is printed on each keycap:
+//  Mar_Rel  +-        1 2 3 4 5 6 7 8 9 0 - #  Backspace  Paper_Up   
+//  Lmar     Tab       Q W E R T Y U I O P ½ [  CRtn       Paper_Down 
+//  Rmar     Lock      A S D F G H J K L ; * ⅔             Micro_Up
+//  T_Set    LShift >< Z X C V B N M , . /?      RShift    Micro_Down
+//  T_Clr              Code  Space               <X        Line_Space (dead)
+
+//  K3N (right shift) is skipped as it is electronically identical to K3B (left shift).
+//  K4P (line space) is skipped as it is not found on the keyboard electronically.
+
+#define LAYOUT(	\
```

Comment:
Layout macros must be configured within info.json, and this file removed.

Reply:
`qmk info -f json` can help with that